packages/sdk/src/index.ts

```diff
@@ -57,7 +57,8 @@ export class MermaidChart {
     this.axios.interceptors.response.use((res: AxiosResponse) => {
       // Reset token if a 401 is thrown
       if (res.status === 401) {
-        this.resetAccessToken();
+        // don't care if this function rejects/resolves
+        void this.resetAccessToken();
```

Comment:
You can remove async from resetAccessToken. The implementation doesn't require it.

Reply:
Fixed by https://github.com/Mermaid-Chart/plugins/pull/13/commits/203aabc922d4e2d43c493e2fb53a8a23496dda33

I was reluctant to make this change, since changing a function to/from `async` is a breaking change, but :shrug:, I think other than https://github.com/Mermaid-Chart/vscode-mermaid-chart/pull/6, nobody is using the `@mermaidchart/sdk` library. Plus, it will be easy for us to bump the version to `^0.2.0`.